### PR TITLE
New Flaky test issue template

### PR DESCRIPTION
**Pull request by lomky**
Based on our BEWG Flaky Test discussion

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/flakey-test-task.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+---
+
+## Description
+<!-- The description should summarize enough information that someone can know what this ticket is about without having to look at information in background/context -->
+
+## Background/context/resources
+<!-- A place for additional information such as links to Slack chats, Sentry alerts, data IDs, research links -->
+
+ - Circle CI Error:  [ <!--CircleCI Failure alert text --> ](<!-- link to circleCI flake -->)
+ - The test has <!-- been skipped --> <!--not been skipped --> in the code
```
Comment (hschallhorn) [inline .github/ISSUE_TEMPLATE/flakey-test-task.md]: Can we change this to a check boxes so they're easy to update if we skip it later down the line?
- [x] Skipped
- [ ] Not skipped

File: .github/ISSUE_TEMPLATE/flakey-test-task.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
+---
+name: Flakey test task
```
Comment (nanotone) [inline .github/ISSUE_TEMPLATE/flakey-test-task.md]: Flaky is misspelled here and in the filename 😛

File: .github/ISSUE_TEMPLATE/flaky-test-task.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+---
+name: Flaky test task
+about: Template for a new task to investigate and fix a flaky test
+title: "[Flaky Test] "
+labels: Flaky test, tech-improvement
+assignees: ''
+
+---
+
+## Description
```
Comment (pkarman) [inline .github/ISSUE_TEMPLATE/flaky-test-task.md]: do we need Description? I think the Background section has the 3 things necessary.

File: .github/ISSUE_TEMPLATE/flaky-test-task.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+<!-- Has our agreed upon default approach for tackling flaky tests. -->
+Time box this investigation and fix.
+
+Remember that if a test has been skipped for a decent amount of time, it may no longer map to the exact code.
+
+If you reach the end of your time box and don't feel like the solution is in sight:
+  - document the work you've done, including dead ends and research
+  - skip the test in the code
+  - file a follow on ticket 
+  - and close this one
```
Comment (ferristseng) [inline .github/ISSUE_TEMPLATE/flaky-test-task.md]: Could this be a checklist too?

File: .github/ISSUE_TEMPLATE/flaky-test-task.md (hunk 3 above)
Comment (lomky) [reply .github/ISSUE_TEMPLATE/flaky-test-task.md]: how about if I add a tag like:

`<!-- Optional: if you have any additional info add it here -->`

**Pull request merged**